Tests/StreamChatUITests/MockViews/MockAvatarView.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+class MockAvatarView: ChatChannelAvatarView {
+    override func updateContent() {
+        super.updateContent()
+        imageView.image = UIImage(named: "pattern1", in: .streamChatUI)
+    }
+}
```

Comment:
This is a workaround for a random image in `ChatChannelAvatarView` when the avatar URL is missing, right? Why don't you just assign it the model object?

Reply:
The model object expects an URL, this way, we won't need a GET Request, and the snapshot will be faster. But maybe we want to test that?

Reply:
It can be a local file URL

Reply:
https://github.com/GetStream/stream-chat-swift/pull/789/commits/f93eaa1d8156efd74bf01b80cd7aaf237e8e2098#r568880558